src/py/flwr/cli/new/templates/app/code/server.pytorch.py.tpl

```diff
@@ -1 +1,130 @@
 """$project_name: A Flower / PyTorch app."""
+
+"""$project_name: A Flower / PyTorch app."""
+
+from collections import OrderedDict
+from typing import Dict, Tuple, List
+
+import torch
+from torch.utils.data import DataLoader
+
+import flwr as fl
+from flwr.common import Metrics
+from flwr.common.typing import Scalar
+
+from datasets import Dataset
+from datasets.utils.logging import disable_progress_bar
+from flwr_datasets import FederatedDataset
+
+from utils import Net, test, apply_transforms
+
+NUM_CLIENTS = 100
```

Comment:
As this might not fit for everyone we ma add the usage to get ENV variables.
If they are not specified we can use these numbers by default.

```python
NUM_CLIENTS = os.getenv(FLWR_NUM_CLIENTS, default = 100)
NUM_ROUNDS = os.getenv(FLWR_NUM_ROUNDS, default = 10)
```